OrbitCore/Profiling.h

```diff
@@ -28,6 +28,10 @@ inline TickType OrbitTicks(uint32_t a_Clock = 1 /*CLOCK_MONOTONIC*/) {
   return 1000000000ll * ts.tv_sec + ts.tv_nsec;
 }
 
+inline uint64_t NanoSecondsFromTicks(TickType start, TickType end) {
```

Comment:
TicksToNanoseconds? or at least s/NanoSeconds/Nanoseconds/

Reply:
Since the other methods in this class are also using the from naming, I would be okay with leaving it as is. Or changing all the names (maybe todo message -> other PR)

Reply:
I changed the name to `TicksToNanoseconds` and also added TODO in the code.